commons/src/main/java/org/infinispan/commons/marshall/Ids.java

```diff
@@ -14,7 +14,8 @@
    int EMPTY_SET = 88;
    int EMPTY_MAP = 89;
    int EMPTY_LIST = 90;
-   // internal collections (id=18 no longer in use, might get reused at a later stage)
+   
+   int IMMUTABLE_LIST = 18;
```

Comment:
The immutable list is quite different than the others, because it must create an array and wrap it instead of using `add(element)`.